fe/fe-core/src/main/java/org/apache/doris/plsql/functions/DorisFunctionRegistry.java

```diff
@@ -196,10 +231,19 @@ public void addUserProcedure(Create_procedure_stmtContext ctx) {
     @Override
     public void save(FuncNameInfo procedureName, String source, boolean isForce) {
         try {
+            SimpleDateFormat f = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+            String createTime = f.format(Calendar.getInstance().getTime());
+            String modifyTime = f.format(Calendar.getInstance().getTime());
```

Comment:
createTime and modifyTime may not be equal because get the current time separately

`String modifyTime=createTime` seems better